rpm/__init__.py

```diff
@@ -6,13 +6,14 @@
 """
 
 import importlib
+import importlib.util
```

Comment:
The import above should have already imported the `util` (sub)module.

Reply:
It (really weirdly) doesn't work unless I import it explicitly:

```
$ python
Python 3.11.7 (main, Aug 23 2024, 00:00:00) [GCC 11.4.1 20231218 (Red Hat 11.4.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import importlib
>>> importlib.util
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'importlib' has no attribute 'util'
```

I am totally perplexed by this.

Reply:
Ah, maybe it's imported in the background by `importlib.reload()` then. Sorry, my bad.